travel-event-host-app/.env

```diff
@@ -0,0 +1,8 @@
+# Dont use this file on development instead create a .env.local
```

Comment:
Suggestion: create a `env.sample` and put the expected environment variables there. 
- And then add `.env` to the `.gitignore`